test/k8sT/Chaos.go

```diff
@@ -228,12 +232,11 @@ var _ = Describe("K8sChaosTest", func() {
 				ctx,
 				helpers.DefaultNamespace,
 				netperfClient,
-				fmt.Sprintf("netperf -l 300 -t TCP_STREAM -H %s", podsIps[netperfServer]))
+				fmt.Sprintf("netperf -l 60 -t TCP_STREAM -H %s", podsIps[netperfServer]))
 
 			restartCilium()
 
 			By("Stopping netperf client test")
-			cancel()
```

Comment:
Unless the double cancel was causing an issue, it would be better to leave it out for another fixups PR.

Reply:
This cancel was killing the kubectl process as opposed to closing the ssh connection (previous version). 

As I wrote above: Killing the process via cancel causes the `kubectl exec netperf` run via os/exec to return error, which causes the test to fail. I am not sure how to stop netperf non-destructively at this point, so I reduced the timeout and simply wait for it to finish. I agree that it's not ideal.